libs/pageserver_api/src/config.rs

```diff
@@ -301,6 +301,8 @@ pub struct TenantConfigToml {
     pub timeline_offloading: bool,
 
     pub wal_receiver_protocol_override: Option<PostgresClientProtocol>,
+
+    pub rel_size_v2_enabled: Option<bool>,
```

Comment:
nit: a comment would be nice, especially to document the behavior when it is enabled and then disabled.